core/src/lib.rs

```diff
@@ -91,9 +95,18 @@ pub type DefaultFileService = FileServiceImpl<
     DefaultFileEncryptionService,
 >;
 
+fn serialize_to_jstring<U: Serialize>(env: &JNIEnv, result: U) -> jstring {
+    let serialized_result =
+        serde_json::to_string(&result).expect("Couldn't serialize result into result string!");
+    env.new_string(serialized_result)
+        .expect("Couldn't create JString from rust string!")
```

Comment:
is this avoidable?

Reply:
In a way it is and it isn't. If I want to not use expects, I would have to get rid of this function and put this in every place it is used, which is a lot. Every JNI or Serde error is handled through this function. Could be other ways to simplify it and not have to use expects

Reply:
But these also kinda need to work, or else I can't return anything.